### PR TITLE
[Ready] Larger branding in the masthead

### PR DESCRIPTION
**Pull request by charypar**
Adding a larger branding to the header.

- [x] Add large FT logo
- [x] mobile treatment
- [x] bring view layer of next-navigation into n-header-footer
- [x] move over the build task for navigation partials, use data from next-navigation as a dependency
- [x] move session tools into menu on smaller screens
- [x] nav stickiness on mobile
- [x] test on stream or article
- [x] fix unused ad slot padding

### CODE REVIEW

File: templates/footer.html [hunk 1]
```diff
@@ -3,6 +3,6 @@
 		<p class="next-footer__back-to-top o-grid-row">
 			<a href="#" data-o-colspan="12" data-trackable="Back to top">Back to top</a>
 		</p>
-		{{>next-navigation/templates/all}}
+		{{> n-header-footer/templates/partials/navigation/all}}
```
Comment (ironsidevsquincy) [inline templates/footer.html]: doesn't `{{> navigation/all}}` work?

Comment (charypar) [inline templates/footer.html]: I don't think `templates/partials` are in partial path

**Comment by charypar** [comment]
@ironsidevsquincy, @adgad can I ask for your :eyes: on this? It's mostly CSS for the new stuff, but I also moved the view part of next-navigation into header-footer and untangled the views of that slightly. The nav templates now get built by n-header-footer and next-navigation only provides the data.

File: src/js/sticky.js [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+
+		requestAnimationFrame(function() {
+			this.scroll();
+		}.bind(this));
+		this.didScroll = false;
+	},
+
+	scroll: function() {
+		// always show on top of the page
+		if(document.body.scrollTop < 50) {
```
Comment (i-like-robots) [inline src/js/sticky.js]: 50 looks like a magic number? Can we get the actual size of the header or maybe specify as a constant at the top of the module?

**Comment by i-like-robots** [comment]
How does this look in core experience? It was somewhat neglected last time around =(

**Comment by ironsidevsquincy** [comment]
Looks good

The building of the nav is still a manual process. Could you add a note in the README (or can we build in circle or pre-commit. Actually how are bower components working in circle ci-land @matthew-andrews)

File: src/js/sticky.js (hunk 2 above)
Comment (charypar) [inline src/js/sticky.js]: It is a bit of a guess, similar to the minimum scroll delta. Not necessarily the height of the header, but definitely depends on it. Will extract to a constant.

**Comment by charypar** [comment]
I just realise there is one more thing I missed on the new one - the white ad slot above isn't used on article and just looks strange. Need to make it only be there when there is an ad (i.e. use the ad for padding rather than the container).

**Comment by charypar** [comment]
@i-like-robots It's very similar, this is really just a facelift. I noticed a bit of a styling issue with JS off (weird?), which I'll fix in a second.

**Comment by charypar** [comment]
@i-like-robots @ironsidevsquincy hopefully addressed all the comments. Otherwise gtg?

**Comment by ironsidevsquincy** [comment]
:+1:

**Pull request merged**